### PR TITLE
User name/email enumaration in - Mautic 

### PR DESCRIPTION
**Pull request by b1nslashsh**
## ✍️ Description
`Mautic` is a Open Source Marketing Automation Software.
Username/email enumaration is possible in forget password function

https://github.com/mautic/mautic

## :male_detective:  Proof of Concept
https://drive.google.com/drive/folders/1qETDHOOBuFDt549FDjbnZulwXWBrL8Mb?usp=sharing
## 💥 Impact
Able to Enumerate users email and username by bruteforce 
due to lack of Ratelimiting
## ✅ Checklist
- [x] _Created and populated the `README.md` and `vulnerability.json` files_
- [x] _Provided the repository URL and any applicable [permalinks]([https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files))_
- [x] _Defined all the applicable weaknesses ([CWEs]([https://cwe.mitre.org/](https://cwe.mitre.org/)))_
- [x] _Proposed the CVSS vector items i.e. User Interaction, Attack Complexity_
- [x] _Checked that the vulnerability affects the latest version of the package released_
- [x] _Checked that a fix does not currently exist that remediates this vulnerability_
- [x] _Complied with all applicable laws_

### CODE REVIEW

**Comment by 418raj** [comment]
Thanks for the disclosure! Unfortunately this isn't in a registry so doesn't meet the disclosure guidelines. I'd recommend attempting to responsibly disclose this to the maintainers of the project though 👍

**Comment by Mik317** [comment]
Hi @418raj :smile:,
I checked and it seems it's in `packagist` even if I'm not sure this is the official one: https://packagist.org/packages/mautic/core

Cheers,
Mik

**Comment by 418raj** [comment]
Ah OK nice one thanks @Mik317

File: bounties/other/mautic/1/vulnerability.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-09-13",
+    "AffectedVersionRange": "*",
+    "Summary": "user name enumaration",
+    "Contributor": {
+        "Discloser": "b1nslashsh",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "Packagist",
```
Comment (JamieSlome) [inline bounties/other/mautic/1/vulnerability.json:11]: Can you please change to `packagist`?

File: bounties/other/mautic/1/vulnerability.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-09-13",
+    "AffectedVersionRange": "*",
+    "Summary": "user name enumaration",
```
Comment (JamieSlome) [inline bounties/other/mautic/1/vulnerability.json:5]: Can you please change to `Username Enumation`?

**Comment by JamieSlome** [comment]
@b1nslashsh - please take a look at the comments above.

Plus, could you change the directory structure to:

`/bounties/other/mautic/core/1/...`?

Thanks! 🍰

**Comment by b1nslashsh** [comment]
@JamieSlome All done :+1:

**Review by Mik317 — APPROVED** [review]
The issue is `valid` but I would lower security :smile: :cake:

Cheers,
Mik

**Comment by JamieSlome** [comment]
@b1nslashsh - your directories need to be `bounties/packagist/...` instead of `bounties/other/...` if the package belongs to the package registry.

Please make the requested changes and we should be good to go, cheers! 🍰

File: bounties/packagist/mautic/core/1/vulnerability.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        "RC": "",
+        "Score": "4.7"
+    },
+    "CVEs": [
+        ""
+    ],
+    "Repository": {
+        "URL": "https://github.com/mautic/mautic",
+        "Codebase": [
+            "JavaScript"
```
Comment (JamieSlome) [inline bounties/packagist/mautic/core/1/vulnerability.json:43]: PHP instead?

**Pull request merged**